README.md

```diff
@@ -28,6 +28,8 @@ This repository contains algorithms and data structures implemented in C# for ed
 			* [Binary GCD](./Algorithms/Numeric/GreatestCommonDivisor/BinaryGreatestCommonDivisorFinder.cs)
 		* [Factorization](./Algorithms/Numeric/Factorization)
 		    * [Trial division](./Algorithms/Numeric/Factorization/TrialDivisionFactorizer.cs)
+		* [Series](./Algorithms/Numeric/Series/)
+	    	* [Maclaurin](./Algorithms/Numeric/Series/Maclaurin.cs)
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6091660)